### ISSUE TITLE
HREX bug patched with Gromacs-2020.2

### ISSUE THREAD

**Issue by AnguseZhang** [plumed/plumed2#605]
Dear plumed team, 
      I tried to use HREX module of Gromacs-2020.2 patched with plumed-2.6, but failed. The errors are as follows. 
![image](https://user-images.githubusercontent.com/38939321/89769309-7caed580-db2f-11ea-9bae-00f8323fd75a.png)
      I used intel/2018.4 , gcc/5.4.0 and cuda/10.0 to compile. I also tried openmpi/3.1.4.      
      If I remove "-hrex" from the command, all things work!
      Could you please have a look? Thanks !

**Comment by trogon44** [comment]
Hi @AnguseZhang, I had the same experience with GROMACS 2020.2 patched with plumed 2.6.1. That is, plumed worked fine for everything else I tested in GROMACS but gave segmentation faults with the -hrex flag specifically at the moment when replica exchange was attempted. When I switched to GROMACS 2019.6 patched with plumed 2.6.1 everything worked as expected.

In the changelog the gromacs 2020-2 patch is listed as being "experimental". It seems like maybe there is a bug with -hrex ?
